wire-ios-sync-engine/Source/UserSession/ZMUserSession.swift

```diff
@@ -825,8 +834,7 @@ extension ZMUserSession: ZMSyncStateDelegate {
                 }
 
                 context.perform {
-                    let service = SupportedProtocolsService(context: context)
-                    service.updateSupportedProtocols()
+                    self?.supportedProtocolsService?.updateSupportedProtocols()
```

Comment:
```suggestion
               let service = SupportedProtocolsService(context: context,
                                                                   oneOnOneResolver: OneOnOneResolver(mlsService: self.mlsService))
               service.updateSupportedProtocols()
```

Reply:
I will keep it as is for now. I feel otherwise it will make the code harder to read in multiple places since we will need to add a default value for the mls service in two places not just one.